paymentintent.go

```diff
@@ -242,7 +219,7 @@ type PaymentIntent struct {
 	Customer                  *Customer                       `json:"customer"`
 	Description               string                          `json:"description"`
 	Invoice                   *Invoice                        `json:"invoice"`
-	LastPaymentError          *PaymentIntentLastPaymentError  `json:"last_payment_error"`
+	LastPaymentError          *Error                          `json:"last_payment_error"`
```

Comment:
Just flagging that it looks like `DocURL` will go away with this change. I think it may just be an oversight that it's not on the standard `Error` struct?

Reply:
Damn nice catch, yes looks like an oversight, let me add this

Reply:
Fixed! 🦅 👁